shared-libs/validation/package-lock.json

```diff
@@ -31,7 +31,124 @@
       }
     },
     "@medic/message-utils": {
```

Comment:
You will need to leave this one as it is:
```
"@medic/message-utils": {
    "version": "file:../message-utils"
}
```
I think it'll give you error when trying to install the project with `npm ci` (which bases the install on the package-lock.json` 
I've sent you some info in slack but pls reach out if you have questions :)